homeassistant/components/elkm1/__init__.py

```diff
@@ -101,7 +101,9 @@ def hostname_from_url(url: str) -> str:
 
 def _host_validator(config: dict[str, str]) -> dict[str, str]:
     """Validate that a host is properly configured."""
-    if config[CONF_HOST].startswith("elks://"):
+    if config[CONF_HOST].startswith("elks://") or config[CONF_HOST].startswith(
+        "elksv1_2://"
+    ):
         if CONF_USERNAME not in config or CONF_PASSWORD not in config:
             raise vol.Invalid("Specify username and password for elks://")
```

Comment:
It looks like this error message also needs updating

Reply:
Thanks!!